ledger/sandbox/src/main/scala/com/digitalasset/platform/configuration/SubmissionConfiguration.scala

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.configuration
+
+import scala.concurrent.duration.{DurationInt, FiniteDuration}
+
+final case class SubmissionConfiguration(
+    maxTtl: FiniteDuration,
+)
+
+object SubmissionConfiguration {
+  lazy val default =
```

Comment:
`object`s are already lazy, since this is the only field it seems redundant.

Reply:
How does such an object interact with the `case class` defined above? Will autogenerated functions (like `SubmissionConfiguration.apply`) be merged with the lazy val here?

Reply:
yeah, that's all fine.

Reply:
Yes.